instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import dummyData from '../../dummy-data';
+import PostsContainer from './PostsContainer';
+import SearchBar from '../SearchBar/SearchBarContainer';
+
+class PostsPage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      posts: [],
+      filteredPosts: []
+    };
+  }
+  componentDidMount() {
+    this.setState({ posts: dummyData });
+  }
+
+  searchPostsHandler = event => {
+    const filteredPosts = this.state.posts.filter(posts =>
+    posts.username === event.target.value);
```

Comment:
You need something to contain and build the search value, right? I'm thinking that if you do it this way, and you search for "Eric," it's only going to search for, "E," then, "r," then, "i," and so on. So I think you want to compare it to some value on state like `searchVal` and accumulate the data in there just like we do with input forms. Does that make sense?